internal/nginx/manager.go

```diff
@@ -34,7 +34,7 @@ const (
 	appProtectAgentStartCmd  = "/opt/app_protect/bin/bd_agent"
 
 	// appPluginParams is the configuration of App-Protect plugin
-	appPluginParams = "tmm_count 4 proc_cpuinfo_cpu_mhz 2000000 total_xml_memory 307200000 total_umu_max_size 3129344 sys_max_account_id 1024 no_static_config"
+	appPluginParams = "tmm_count 4 proc_cpuinfo_cpu_mhz 2000000 total_xml_memory 471859200 total_umu_max_size 3129344 sys_max_account_id 1024 no_static_config"
```

Comment:
What happens if this is more than the available memory in the pod?

Reply:
Need to try this out. probably BD will crash